### PR TITLE
Rename bioconda curl

### PR DESCRIPTION
**Pull request by dpryan79**
@johanneskoester Can you have a look? This will hopefully work and is now using the same version of curl that comes with the most recent version of anaconda (this will hopefully avoid other issues).

### CODE REVIEW

File: recipes/pybigwig/0.1.11/meta.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ source:
 
 build:
   number: 0
+  skip: True # [osx]
```
Comment (johanneskoester) [inline recipes/pybigwig/0.1.11/meta.yaml]: Why the skip here if it is added of osx-whitelist above?

Comment (johanneskoester) [inline recipes/pybigwig/0.1.11/meta.yaml]: Oh, is this only for this specific version of the package?

Comment (dpryan79) [inline recipes/pybigwig/0.1.11/meta.yaml]: Yup, it's an old version that never existed in OSX before.

Comment (johanneskoester) [inline recipes/pybigwig/0.1.11/meta.yaml]: ok!

**Pull request merged**